code/modules/projectiles/projectile/rocket.dm

```diff
@@ -39,77 +45,55 @@
 /obj/item/projectile/rocket/lowyield
 	name = "low yield rocket"
 	icon_state = "rpground"
-	explosive = 0
 	damage = 45
 	stun = 10
 	weaken = 10
-	damage_type = BRUTE
-	nodamage = 0
-	flag = "bullet"
-
-/obj/item/projectile/rocket/lowyield/to_bump(var/atom/A)
-	explosion(A, -1, 0, 3, 5) //RPGs pack a serious punch and will cause massive structural damage in your average room, but won't punch through reinforced walls
-	..()
-	if(!gcDestroyed)
-		qdel(src)
+	exdev 	= -1
+	exheavy = 0
+	exlight = 3
+	exflash = 5
 
 /obj/item/projectile/rocket/blank
 	name = "blank rocket"
-	icon_state = "rpground"
-	explosive = 0
 	damage = 5
-	stun = 5
 	weaken = 10
 	agony = 10
-	damage_type = BRUTE
-	nodamage = 0
-	flag = "bullet"
-
-/obj/item/projectile/rocket/blank/to_bump(var/atom/A)
-	explosion(A, -1, 0, 0, 0)
-	..()
-	if(!gcDestroyed)
-		qdel(src)
+	exdev 	= -1
+	exheavy = 0
+	exlight = 0
+	exflash = 0
 
-/obj/item/projectile/rocket/emp
+/obj/item/projectile/rocket/blank/emp
 	name = "EMP rocket"
-	icon_state = "rpground"
-	explosive = 0
 	damage = 10
-	stun = 5
-	weaken = 10
 	agony = 30
-	damage_type = BRUTE
-	nodamage = 0
-	flag = "bullet"
+	emheavy = 3
+	emlight = 5
 
 /obj/item/projectile/rocket/emp/to_bump(var/atom/A)
-	explosion(A, -1, 0, 0, 0)
 	empulse(A, 3, 5)
 	..()
-	if(!gcDestroyed)
-		qdel(src)
-
-/obj/item/projectile/rocket/stun
+	
+/obj/item/projectile/rocket/blank/stun
 	name = "stun rocket"
-	icon_state = "rpground"
-	explosive = 0
 	damage = 15
 	stun = 20
 	weaken = 20
 	agony = 30
-	damage_type = BRUTE
-	nodamage = 0
-	flag = "bullet"
 
 /obj/item/projectile/rocket/stun/to_bump(var/atom/A)
-	explosion(A, -1, 0, 0, 0)
 	flashbangprime(TRUE, FALSE, FALSE)
 	..()
-	if(!gcDestroyed)
-		qdel(src)
-
-/obj/item/projectile/nikita
+		
+/obj/item/projectile/rocket/lowyield/extreme
```

Comment:
>low yield

big hmm

Reply:
it's already coded/mapped in and i didn't feel like changing it and that specific subtype explodes